### PR TITLE
basic html testing across all potential fields

### PR DESCRIPTION
**Pull request by lmd59**
Pull requests into cqm-parsers require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [ ] This pull request describes why these changes were made.
- [ ] Internal ticket for this PR:
- [ ] Internal ticket links to this PR
- [ ] Code diff has been done and been reviewed
- [ ] Tests are included and test edge cases
- [ ] Tests have been run locally and pass

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: test/unit/html/patient_round_trip_test.rb [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+      assert dt.respond_to?(field), "datatype generation discrepancy, should contain field #{field}"
+      check_for_type(html, dt, field)
+    end
+
+    def check_for_type(html, dt, field)
+      attr = dt.send(field)
+      if attr.respond_to?(:strftime)
+        # timey object
+        formatted_date = attr.localtime.strftime('%FT%T')
+        assert html.include?(formatted_date), "html should include date/time value #{formatted_date}"
```
Comment (dczulada) [inline test/unit/html/patient_round_trip_test.rb:68]: Could these assertions also include the datatype and field

for example:

"html should include date/time value #{formatted_date} for #{field} in #{dt._type}"

Might make debugging easier

**Review by dczulada — CHANGES_REQUESTED** [review]
Looks great, 1 minor change.

**Review by dczulada — APPROVED** [review]
